logging_format/api.py

```diff
@@ -0,0 +1,38 @@
+"""
+Flake8 entry point.
+
+"""
+from logging_format.visitor import LoggingVisitor
+from logging_format.whitelist import Whitelist
+
+
+__version__ = "0.1.0"
+
+
+class LoggingFormatValidator(object):
```

Comment:
The interface for `flake8` extension is [described here](http://flake8.pycqa.org/en/2.5.5/extensions.html).

It's not super detailed, but it's enough.